doc/source/user_guide/cookbook.rst

```diff
@@ -242,7 +242,7 @@ Ambiguity arises when an index consists of integers with a non-zero start or non
 New columns
 ***********
 
-`Efficiently and dynamically creating new columns using applymap
+`Efficiently and dynamically creating new columns using DataFrame.map (previously named applymap)
```

Comment:
Why have "previously named" here?

Reply:
It is because the linked stackoverflow article is older than the method name change, so clears up the relationship between DataFrame.map in pandas now and DataFrame.applymap in the article.